manifests/config/exporter/exporter.yaml

```diff
@@ -52,7 +52,17 @@ spec:
             memory: "400Mi"
             cpu: "100m"
         securityContext:
-          privileged: true
+           runAsUser: 2000
```

Comment:
I followed update here and seems I can setup it locally 

```
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
uid=2000(2000) gid=2000 groups=2000
```

so the TODO for this is to make our CI able to run with non-root user ,but you encounter some problem ,correct?

Reply:
yes, non-privileged root needs to have 1M memlock (ulimit -l 10240) to run ebpf. But Github Action cannot set it. So we have to have a kustomize for non root user. @jichenjc can you take this into a new PR?